src/lib/models/event-history/get-event-categorization.test.ts

```diff
@@ -142,7 +150,10 @@ describe('Event Category Data Structures', () => {
   });
 });
 
-const categories: Record<EventTypeCategory, EventType[]> = {
+const categories: Record<
+  Exclude<EventTypeCategory, 'local-activity'>,
```

Comment:
I've opted not to touch any of the logic in `getEventCategory`, etc. (and just omit the category here), but if we want to be categorizing events as `LocalActivity`, outside of just filtering for them, we may want to revisit this.